exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "562eac65-b205-49d4-971e-9ed0090796b4",
+   "metadata": {},
+   "source": [
+    "#### Package Lists\n",
+    "The package lists have a unified format. Each line consists of the package name and the package version separated by \"|\", e.g `xgboost|1.3.3`. You can comment out a whole line by adding\"#\" at the beginning. You can also add a trailing comment to a package definition by adding a \"#\" after the package definition. We usually recommend to pin the version, such that there are no surprises for which version gets installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce88261d-efc4-470e-a875-447a28c0125c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "show_files([slctmanager.custom_pip_file])\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5857a6fb-f0b7-413f-8ce5-88ced8645abb",
+   "metadata": {},
+   "source": [
+    "We are now going to append the \"xgboost\" Python package to one of the package lists by adding `xgboost|2.0.3` and `scikit-learn|1.5.0` to the `flavor_customization/packages/python3_pip_packages` file. \n",
+    "Notes:\n",
+    " - running the following command multiple times will iteratively append the packages\n",
+    " - you can also click on the link and modify the file directly\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe3db39f-5588-44a9-9e64-d4ec495423ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xgboost_pkg = slct_manager.PipPackageDefinition(pkg=\"xgboost\", version=\"2.0.3\")\n",
+    "scikit_learn_pkg = slct_manager.PipPackageDefinition(pkg=\"scikit-learn\", version=\"1.5.0\")\n",
+    "slctmanager.append_custom_packages([xgboost_pkg, scikit_learn_pkg])\n",
+    "show_files([slctmanager.custom_pip_file])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "834a2314-13ab-4ae8-bca5-9d1861376419",
+   "metadata": {},
+   "source": [
+    "#### Rebuilding the customized Flavor\n",
+    "After changing the flavor you need to rebuild it. You can do it by running `export` again. Exaslct automatically recognizes that the flavor has changed and builds a new version of the container."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e7e9bbb-bf77-474f-949f-9521e9e3fd86",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slctmanager.export()\n",
+    "print(\"Ready\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0cec469-33de-4b1a-b481-735e69722c9f",
+   "metadata": {},
+   "source": [
+    "Note: Your old container doesn't get lost, because when you change a flavor your container gets a new hash code. If you revert your changes the system automatically uses the existing cached container. Below you can see the content of the cache directory for the containers."
```

Comment:
Also here, the term "flavor" doesn't sound right. 
Or is flavor a synonym for "container specification"?